package.json

```diff
@@ -115,9 +121,11 @@
     "sass": "^1.52.0",
     "serverless-mysql": "^1.5.4",
```

Comment:
Will you replace the `serverless-mysql` implementation with @prisma and @trpc now to avoid using two different architectures in our project which make maintenance more difficult in the future.

Reply:
@norbertschuler Yes. That is the plan, once my-forest is completed. But it will be done in another PR.

Reply:
@Shreyaschorge Is this done, or has an issue been created to work on this later?